### PR TITLE
fix test:LoadBalanceEndpointTestCase

### PR DESCRIPTION
**Pull request by sajithaliyanage**

### CODE REVIEW

File: integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/LoadBalanceEndpointTestCase.java [hunk 1]
```diff
@@ -99,122 +68,49 @@ public void close() throws Exception {
...
 
         response = lbClient.sendLoadBalanceRequest(getProxyServiceURLHttp("loadbalancingEndPoint"), null);
         Assert.assertNotNull(response);
-        Assert.assertTrue(response.toString().contains("Response from server: Server_2"));
+        Assert.assertTrue(response.contains("Response from server: Server_2"));
 
         response = lbClient.sendLoadBalanceRequest(getProxyServiceURLHttp("loadbalancingEndPoint"), null);
         Assert.assertNotNull(response);
-        Assert.assertTrue(response.toString().contains("Response from server: Server_3"));
+        Assert.assertTrue(response.contains("Response from server: Server_3"));
```
Comment (a5anka) [inline integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/LoadBalanceEndpointTestCase.java]: Shall we assert messages?

File: integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/LoadBalanceEndpointTestCase.java [hunk 2]
```diff
@@ -99,122 +68,49 @@ public void close() throws Exception {
...
-
-        resourceAdminServiceClient
-                .addCollection("/_system/config/", "test_ep_config", "", "Contains test Default EP files");
-        resourceAdminServiceClient
-                .addResource("/_system/config/test_ep_config/loadbalancingEP_Test.xml", "application/xml", "xml files",
-                        new DataHandler(new URL("file:///" + getClass().getResource(
-                                "/artifacts/ESB/endpoint/loadbalancingEndpointConfig/loadbalancingEP_Test.xml")
-                                .getPath())));
-        Thread.sleep(1000);
+        Assert.assertTrue(response.contains("Response from server: Server_3"));
```
Comment (a5anka) [inline integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/LoadBalanceEndpointTestCase.java]: Shall we add assert messages?

File: integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/LoadBalanceEndpointTestCase.java (hunk 1 above)
Comment (sajithaliyanage) [reply integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/LoadBalanceEndpointTestCase.java]: Done in https://github.com/wso2/micro-integrator/pull/574

File: integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/LoadBalanceEndpointTestCase.java (hunk 2 above)
Comment (sajithaliyanage) [reply integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/LoadBalanceEndpointTestCase.java]: Done in https://github.com/wso2/micro-integrator/pull/574

**Pull request merged**